### ISSUE TITLE
Results and comparison

### ISSUE THREAD

**Issue by kinalmehta** [hr0nix/optax-adan#1]
Hi,

Thanks for the awesome work.

Do you mind sharing comparison results of adan vs adam/adamW on any task using this implementation?

Thanks
Kinal

**Comment by hr0nix** [comment]
Hello!

There's a comparison with Adam on a number of test functions in the [accompanying notebook](https://colab.research.google.com/drive/19--gju3ELQ9qPbDZbE4NmEnGBLJC901x?usp=sharing)

Or are you looking for something more realistic, like training a neural net?

**Comment by kinalmehta** [comment]
Yeah, I was looking for a comparison on any of the tasks mentioned in the paper.

Maybe I could set it up and send a PR with the results.

**Comment by hr0nix** [comment]
> Maybe I could set it up and send a PR with the results.

Contributions are very welcome! If that's something you'd be interested in doing, that would be awesome!
